packages/evm/package.json

```diff
@@ -56,6 +56,7 @@
     "visualize:bundle": "npx vite build --config=./vite.config.bundler.ts --emptyOutDir=false --outDir ."
   },
   "dependencies": {
+    "@ethereumjs/block": "^5.2.0",
```

Comment:
Oh, this is very very very unlucky, and might become a blocker for the whole PR.

We fought really hard to get Block out of EVM, e.g. by adding a `defaultBlock()` method faking a default block. It would definitely not be worth it to re-add just for this refactor. This would bloat the (non-tree-shaked) bundle size and so this would be an improved-on-one-end-worsened-on-the-other situation. 

Scratching my head around this, but haven't found a good solution yet. 🤔

Reply:
Update: we could pass in the method `cliqueSigner` as an `EVMOpts` and check on instantiation - if clique is active (via Common) - that the option is passed (and otherwise throw).

Not fully sure if its worth it but also would go along if you decide to do. Maybe it's on the "ok" side. 

Yeah, maybe it's worth it. Go ethereum has its clique code also still in https://github.com/etclabscore/core-geth/blob/master/consensus/clique/clique.go , I would also really like to keep, othewise we are so super-dependent on the Consensus side of things for "everything" [tm].

And at the end not so much work, just a simple option-addition.